build.sbt

```
@@ -188,6 +191,7 @@ sparkRunJobFlowRequest := sparkRunJobFlowRequest.value
   .withTags(new Tag("Project Lead", "Thomas Maschler"))
   .withTags(new Tag("Name", "geotrellis-treecoverloss"))
 sparkEmrConfigs := List(
+  // reference to example by geotrellis: https://github.com/geotrellis/geotrellis-spark-job.g8/blob/master/src/main/g8/build.sbt#L70-L91
```

In the example they run a bootstrap script. How do we get the GDAL drivers onto the worker nodes?

---

I believe the this would be
```suggestion
sparkEmrBootstrap := List(
    BootstrapAction(
      "Install GDAL 2.4 dependencies",
      "s3://geotrellis-build-artifacts/rpms/bootstrap.sh",
      "s3://geotrellis-build-artifacts", "gdal-2.4.1"
    )
)
```